extensions/wikia/AdEngine/js/video/player/porvata/googleImaPlayerFactory.js

```diff
@@ -219,7 +228,8 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory', [
 			reload: reload,
 			removeEventListener: removeEventListener,
 			resize: resize,
-			setAutoPlay: setAutoPlay
+			setAutoPlay: setAutoPlay,
+			viewMode: imaViewMode
```

Comment:
I think we don't need to export this property. When calling `ima.resize` from `porvataPlayerFactory.js` we just can pass true/false for fullscreen option, and don't care about imaViewMode in there

Reply:
i think that the viewport enum is more descriptive in this case